lib/index.ts

```diff
@@ -455,6 +455,7 @@ export class Worker {
 
 		// Get all contracts provided by plugins
 		const pluginContracts = this.pluginManager.getCards();
+		logger.info(logContext, 'FIXME LOAD RELATIONSHIPS FROM PLUGINS');
```

Comment:
I think we can just treat these the same as any other contracts and not add any special logic